src/examples/src/widgets/date-input/Required.tsx

```diff
@@ -0,0 +1,24 @@
+import { create, tsx } from '@dojo/framework/core/vdom';
+import DateInput from '@dojo/widgets/date-input';
+import icache from '@dojo/framework/core/middleware/icache';
+import Example from '../../Example';
+
+const factory = create({ icache });
+
+export default factory(function Controlled({ middleware: { icache } }) {
```

Comment:
Can you rename this to `Required` please

Reply:
yup!